pkg/ghclient/wrappers.go

```diff
@@ -143,10 +143,10 @@ func (c *Client) ForEachPR(owner, repo string, opts *github.PullRequestListOptio
 
 // GetCollaborators returns all github users who are members or outside collaborators of the repo.
 func (c *Client) GetCollaborators(org, repo string) ([]*github.User, error) {
-	opts := &github.ListOptions{}
+	opts := &github.ListCollaboratorsOptions{}
 	collaborators, err := c.depaginate(
 		fmt.Sprintf("getting collaborators for '%s/%s'", org, repo),
-		opts,
+		&opts.ListOptions,
```

Comment:
nit: The changes to this function seem unnecessary. Can't `github.ListOptions` be used directly as it was previously?

Reply:
`ListCollaborators()` (L151) now takes a `ListCollaboratorsOptions` instead of a `ListOptions` (since v10 - https://github.com/google/go-github/pull/706).